lib/bolt/target.rb

```diff
@@ -66,7 +66,7 @@ def safe_name
     end
 
     def target_alias
-      inventory_target.target_alias
+      inventory_target.target_aliases
```

Comment:
I think the intent of the `Inventory::Target` is to handle aliases in the instance variable https://github.com/puppetlabs/bolt/blob/17d88e7c3ae1eece83327be5dcf7a3541c41e2b6/lib/bolt/inventory/target.rb#L39 I implemented the `target_aliases` method on v1 inventory because we do not track that per target. I think we need to figure out how to make storing aliases on instance variables (or if that is impossible we need to delete them).